From 6707d64dacf4beab96a2031363360c7404d28bb0 Mon Sep 17 00:00:00 2001
Message-Id: <6707d64dacf4beab96a2031363360c7404d28bb0.1349175436.git.minovotn@redhat.com>
In-Reply-To: <94968b7fa9b14e71f004474d7ce77e189e6a2bf3.1349175436.git.minovotn@redhat.com>
References: <94968b7fa9b14e71f004474d7ce77e189e6a2bf3.1349175436.git.minovotn@redhat.com>
From: Eduardo Habkost <ehabkost@redhat.com>
Date: Fri, 28 Sep 2012 16:50:14 +0200
Subject: [PATCH 10/34] target-i386: move compatibility static variables to
 top of file

RH-Author: Eduardo Habkost <ehabkost@redhat.com>
Message-id: <1348851023-31907-10-git-send-email-ehabkost@redhat.com>
Patchwork-id: 42516
O-Subject: [RHEL6 qemu-kvm PATCH 09/18] target-i386: move compatibility static variables to top of file
Bugzilla: 833152
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Igor Mammedov <imammedo@redhat.com>
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>

Bugzilla: 833152
Upstream status: not applicable

More variables will be added, so it's better to keep them all at the
same place.

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 target-i386/cpuid.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 target-i386/cpuid.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/target-i386/cpuid.c b/target-i386/cpuid.c
index c18f9b2..d61f329 100644
--- a/target-i386/cpuid.c
+++ b/target-i386/cpuid.c
@@ -87,6 +87,10 @@ typedef struct model_features_t {
 int check_cpuid = 0;
 int enforce_cpuid = 0;
 
+/* machine-type compatibility settings: */
+static bool kvm_pv_eoi_disabled;
+static bool cpuid_leaf10_disabled;
+
 static void host_cpuid(uint32_t function, uint32_t count, uint32_t *eax,
                        uint32_t *ebx, uint32_t *ecx, uint32_t *edx);
 
@@ -811,8 +815,6 @@ static int check_features_against_host(CPUX86State *env, x86_def_t *guest_def)
     return rv;
 }
 
-static bool kvm_pv_eoi_disabled;
-
 static int cpu_x86_find_by_name(x86_def_t *x86_cpu_def, const char *cpu_model)
 {
     unsigned int i;
@@ -1121,8 +1123,6 @@ static void get_cpuid_vendor(CPUX86State *env, uint32_t *ebx,
     }
 }
 
-static bool cpuid_leaf10_disabled;
-
 void cpu_x86_cpuid(CPUX86State *env, uint32_t index, uint32_t count,
                    uint32_t *eax, uint32_t *ebx,
                    uint32_t *ecx, uint32_t *edx)
-- 
1.7.11.4

